src/SBInclinedSersic.cpp

```diff
@@ -0,0 +1,470 @@
+    }
+
+    double SBInclinedSersic::SBInclinedSersicImpl::xValue(const Position<double>& p) const
+    {
+        throw std::runtime_error(
+            "Real-space expression of SBInclinedSersic is not yet implemented.");
+        return 0;
+    }
+
+    std::complex<double> SBInclinedSersic::SBInclinedSersicImpl::kValue(const Position<double>& k) const
+    {
+        double kx = k.x*_r0;
+        double ky = k.y*_r0;
+        return _flux * kValueHelper(kx,ky);
+    }
+
+    void SBInclinedSersic::SBInclinedSersicImpl::fillKImage(ImageView<std::complex<double> > im,
+                                                double kx0, double dkx, int izero,
+                                                double ky0, double dky, int jzero) const
+    {
+        dbg<<"SBInclinedSersic fillKImage\n";
+        dbg<<"kx = "<<kx0<<" + i * "<<dkx<<", izero = "<<izero<<std::endl;
+        dbg<<"ky = "<<ky0<<" + j * "<<dky<<", jzero = "<<jzero<<std::endl;
+        if (izero != 0 || jzero != 0) {
+            xdbg<<"Use Quadrant\n";
+            fillKImageQuadrant(im,kx0,dkx,izero,ky0,dky,jzero);
+        } else {
+            xdbg<<"Non-Quadrant\n";
+            const int m = im.getNCol();
+            const int n = im.getNRow();
+            std::complex<double>* ptr = im.getData();
+            int skip = im.getNSkip();
+            assert(im.getStep() == 1);
+
+            kx0 *= _r0;
+            dkx *= _r0;
+            ky0 *= _r0;
+            dky *= _r0;
+
+            for (int j=0; j<n; ++j,ky0+=dky,ptr+=skip) {
+                double kx = kx0;
+                for (int i=0;i<m;++i,kx+=dkx)
+                    *ptr++ = _flux * kValueHelper(kx,ky0);
+            }
+        }
+    }
+
+    void SBInclinedSersic::SBInclinedSersicImpl::fillKImage(ImageView<std::complex<double> > im,
+                                                double kx0, double dkx, double dkxy,
+                                                double ky0, double dky, double dkyx) const
+    {
+        dbg<<"SBInclinedSersic fillKImage\n";
+        dbg<<"kx = "<<kx0<<" + i * "<<dkx<<" + j * "<<dkxy<<std::endl;
+        dbg<<"ky = "<<ky0<<" + i * "<<dkyx<<" + j * "<<dky<<std::endl;
+        const int m = im.getNCol();
+        const int n = im.getNRow();
+        std::complex<double>* ptr = im.getData();
+        int skip = im.getNSkip();
+        assert(im.getStep() == 1);
+
+        kx0 *= _r0;
+        dkx *= _r0;
+        dkxy *= _r0;
+        ky0 *= _r0;
+        dky *= _r0;
+        dkyx *= _r0;
+
+        for (int j=0; j<n; ++j,kx0+=dkxy,ky0+=dky,ptr+=skip) {
+            double kx = kx0;
+            double ky = ky0;
+            for (int i=0; i<m; ++i,kx+=dkx,ky+=dkyx)
+                *ptr++ = _flux * kValueHelper(kx,ky);
+        }
+    }
+
+    double SBInclinedSersic::SBInclinedSersicImpl::maxK() const
+    {
+        return _maxk * _inv_r0;
+    }
+    double SBInclinedSersic::SBInclinedSersicImpl::stepK() const
+    {
+        double stepk = _info->stepK() * _inv_re;
```

Comment:
The problem I was having with _inv_r0 was that it was excluding too much flux outside of 1/stepK (even in the face-on case, when it matches the Sersic profile - it looks like that profile was never tested for this), so it actually needs to be somewhat smaller than what the SersicInfo class is giving. I'm looking into this a bit more now and update soon; it might be that the class's estimate of stepK is what needs to be corrected